tests/unit/configuration_subsystem/test_posix_message_queue.py

```diff
@@ -0,0 +1,36 @@
+"""Tests related to a missing ``/dev/mqueue`` directory when using ``podman``."""
+from pathlib import Path
+
+
+def exists(_arg):
```

Comment:
maybe rename this to

```suggestion
def _never_exists(_arg):
```

Reply:
tests were completely rewritten....

Reply:
I'd stick with this function and not fall into the trap of stdlib unittest.

Reply:
This isn't really relevant after the last push. Let me know